src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+public class Task {
```

Comment:
I think this class is well written! Just need to have javadocs for the class and javadocs for the method to make it perfect!